client/src/components/Picture/ProfilePic.js

```diff
@@ -9,12 +9,14 @@ const InitialDiv = styled.div`
   margin: auto;
   margin-bottom: 1rem;
   border-radius: 50%;
-  border: ${(props) => (props.hasPhoto ? "none" : "0.2rem solid #425af2;")}
+  border: none;
+  border: ${(props) => (props.hasPhoto ? "none" : "none")}
```

Comment:
you have a photo or not border is none right? so do we need this line?

Reply:
Yes, I will clean it